demo/demo_mot.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+import tempfile
+from argparse import ArgumentParser
+
+import mmcv
+import mmengine
+
+from mmdet.apis import inference_mot, init_track_model
+from mmdet.registry import VISUALIZERS
+from mmdet.utils import register_all_modules
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('config', help='config file')
+    parser.add_argument('--input', help='input video file or folder')
```

Comment:
Please refer to another demo script to modify these arguments to keep the same using custom as much as possible.